vertical-pod-autoscaler/e2e/update-kubernetes-deps.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
Why not use https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/hack/update-kubernetes-deps-in-e2e.sh ?

Reply:
Yeah, I delete this file. It's a copy.